docs/fundamentals/toc.yml

```diff
@@ -358,10 +358,577 @@ items:
+          - name: CA5398
+            href: code-analysis/rules/ca5398.md
+          - name: CA5399
+            href: code-analysis/rules/ca5399.md
+          - name: CA5400
+            href: code-analysis/rules/ca5400.md
+          - name: CA5401
+            href: code-analysis/rules/ca5401.md
+          - name: CA5402
+            href: code-analysis/rules/ca5402.md
+          - name: CA5403
+            href: code-analysis/rules/ca5403.md
+        - name: Usage rules
+          items:
+          - name: Overview
+            displayName: "usage rules"
+            href: code-analysis/rules/usage-warnings.md
+          - name: CA1801
+            href: code-analysis/rules/ca1801.md
+          - name: CA1816
+            href: code-analysis/rules/ca1816.md
+          - name: CA2200
+            href: code-analysis/rules/ca2200.md
+          - name: CA2201
+            href: code-analysis/rules/ca2201.md
+          - name: CA2207
+            href: code-analysis/rules/ca2207.md
+          - name: CA2208
+            href: code-analysis/rules/ca2208.md
+          - name: CA2211
+            href: code-analysis/rules/ca2211.md
+          - name: CA2213
+            href: code-analysis/rules/ca2213.md
+          - name: CA2214
+            href: code-analysis/rules/ca2214.md
+          - name: CA2215
+            href: code-analysis/rules/ca2215.md
+          - name: CA2216
+            href: code-analysis/rules/ca2216.md
+          - name: CA2217
+            href: code-analysis/rules/ca2217.md
+          - name: CA2219
+            href: code-analysis/rules/ca2219.md
+          - name: CA2225
+            href: code-analysis/rules/ca2225.md
+          - name: CA2226
+            href: code-analysis/rules/ca2226.md
+          - name: CA2227
+            href: code-analysis/rules/ca2227.md
+          - name: CA2229
+            href: code-analysis/rules/ca2229.md
+          - name: CA2231
+            href: code-analysis/rules/ca2231.md
+          - name: CA2234
+            href: code-analysis/rules/ca2234.md
+          - name: CA2235
+            href: code-analysis/rules/ca2235.md
+          - name: CA2237
+            href: code-analysis/rules/ca2237.md
+          - name: CA2241
+            href: code-analysis/rules/ca2241.md
+          - name: CA2242
+            href: code-analysis/rules/ca2242.md
+          - name: CA2243
+            href: code-analysis/rules/ca2243.md
+          - name: CA2244
+            href: code-analysis/rules/ca2244.md
+          - name: CA2245
+            href: code-analysis/rules/ca2245.md
+          - name: CA2246
+            href: code-analysis/rules/ca2246.md
+          - name: CA2247
+            href: code-analysis/rules/ca2247.md
+          - name: CA2248
+            href: code-analysis/rules/ca2248.md
+          - name: CA2249
+            href: code-analysis/rules/ca2249.md
+      - name: Code style rules
+        href: /visualstudio/ide/editorconfig-language-conventions
```

Comment:
Ah, I presume this will not be reachable from preview. That seems reasonable. This enables us to start creating IDE style doc pages here. @gewarren Let me know if you were planning to do that work, otherwise, I can start creating initial set of IDE rule doc pages here and then we can ask Roslyn IDE team members to spread out writing these docs.

Reply:
Yes, I verified that if you change ?branch=pr-en-us-20702 to ?branch=master, the link works. It's because the pr-en-us-20702 branch doesn't exist in the VS repo.

I was planning to do the work if no-one else had signed up, but to be honest, I have a lot of other docs to work on too, so any help would be appreciated! Thank you!

Reply:
Sure, I can work on this today.